project.js

```diff
@@ -2,59 +2,17 @@ const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
 function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-
-function randomInteger(min, max) {
-    let range = max - min + 1;
-    return min + Math.floor(Math.random() * range);
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
 }
 
 start();
 
-function getRandomInt(max) {
-    return Math.floor(Math.random() * Math.floor(max));
-}
```

Comment:
A random number in a range is good. A binary search (choosing the mean of the range each time) is better.